en/analyze_view/mavlink_inspector.md

```diff
@@ -0,0 +1,36 @@
+# MAVLink Inspector Widget
```

Comment:
@dogmaphobic This is update to MAVLink inspector docs. Can you please sanity check.